tests/Aspire.Hosting.Azure.Tests/AzureFunctionsTests.cs

```diff
@@ -106,6 +105,7 @@ private sealed class TestProject : IProjectMetadata
                 ["funcapp"] = new()
                 {
                     CommandLineArgs = "--port 7071",
+                    LaunchBrowser = false,
```

Comment:
?

Reply:
This came because I did the revert [here](https://github.com/dotnet/aspire/pull/5873/commits/cbd20d582300759c81c8219360aa058be7f5d8be). The original test data set `LaunchBrowser = false` for no reason really. :/